src/main/java/org/bytedeco/javacpp/tools/ClearMojo.java

```diff
@@ -40,7 +40,7 @@ public class ClearMojo extends AbstractMojo {
         try {
             Loader.clearCacheDir();
         } catch (IOException e) {
-            throw new MojoExecutionException(e);
```

Comment:
`MojoExecutionException(java.lang.Throwable)` is available since 3.8.3. https://github.com/apache/maven/blob/84538c9988a25aec085021c365c560670ad80f63/maven-plugin-api/src/main/java/org/apache/maven/plugin/MojoExecutionException.java#L85-L87

Reply:
That's not important, we can just use e.toString() or something, but is that the only issue?

Reply:
Yes, it seems so.